main.js

```diff
@@ -0,0 +1,420 @@
+
+
+// Submit Button
+const submitButton = document.getElementById('submit-btn');
+
+submitButton.addEventListener('click', showResults);
+
+// let resultPage = document.createElement('results');
+
+
+function showResults() {
+  let resultPage = document.getElementById('results');
+  document.getElementById('quiz').style.display = 'none';
+
+  let headerResults = document.createElement('h1');
+  let text = document.createTextNode('Your results:');
+  headerResults.appendChild(text);
+  resultPage.appendChild(headerResults);
+
+  let rightAnswers = document.createElement('p');
+  let textAns = document.createTextNode(
+    `${pointsCounter} out of ${questions.length}`
+  );
+  rightAnswers.appendChild(textAns);
+  resultPage.appendChild(rightAnswers);
+
+  let enteredPoints = `${pointsCounter}`;
+
+  let totalPoints = questions.length;
+  let percent = (enteredPoints * 100) / totalPoints;
+
+  let userPercent = document.createElement('p');
+  let userPercentElement = document.createTextNode(percent + ' %');
+  userPercent.appendChild(userPercentElement);
+  resultPage.appendChild(userPercent);
+
+
+  if (percent >= 80) {
+    let p1 = document.createElement('p');
+    let text1 = document.createTextNode('Impressive. You did an amazing job!');
+    p1.appendChild(text1);
+    resultPage.appendChild(p1);
+    
+    let img1 = document.createElement('img'); 
+    img1.setAttribute(
+      'src',
+      'https://media.giphy.com/media/YRuFixSNWFVcXaxpmX/giphy.gif'
+    );
+    img1.setAttribute('alt', 'img1');
+    let src1 = document.getElementById('results');
+
+    src1.appendChild(img1);
+  } else if (percent >= 45) {
+    let p2 = document.createElement('p');
+    let text2 = document.createTextNode('Well, I think you can do better…');
+    p2.appendChild(text2);
+    resultPage.appendChild(p2);
+
+    let img2 = document.createElement('img'); 
+    img2.setAttribute(
+      'src',
+      'https://media.giphy.com/media/qLWdMYX1NYF2g/giphy.gif'
+    );
+    img2.setAttribute('alt', 'img2');
+    let src2 = document.getElementById('results');
+    src2.appendChild(img2);
+  } else {
+    let p3 = document.createElement('p');
+    let text3 = document.createTextNode('Loser! Please try again.');
+    p3.appendChild(text3);
+    resultPage.appendChild(p3);
+
+    let img3 = document.createElement('img'); // <img> <img1>
+    img3.setAttribute(
+      'src',
+      'https://media.giphy.com/media/pGYmjOL2vDiak/giphy.gif'
+    );
+    img3.setAttribute('alt', 'img3');
+    let src3 = document.getElementById('results');
+    src3.appendChild(img3);
+  }
+
+  // create a div for the two buttons on the resultpage
+  let buttonsResultPage = document.createElement('div');
+  resultPage.appendChild(buttonsResultPage);
+
+  let checkYourAnswersButton = document.createElement('BUTTON');
+  checkYourAnswersButton.setAttribute('id', 'check-answers-btn');
+  let text1 = document.createTextNode('Check your answers');
+
+  checkYourAnswersButton.appendChild(text1);
```

Comment:
I think in most cases it's more readable if you use `checkYourAnswersButton.innerText = 'Check your answers'` (it's just one line then) versus appending a text node